docs/docs/firmware/state-monitor-esp32.md

```diff
@@ -53,17 +53,17 @@ The message payload should be JSON and it's format is defined in a JSON schema d
 See the `config` value in the `/adopt` payload.
 
 ### Default Input Type
-By default all inputs are initialised as type `toggle`. Individual inputs can then be configured as required (see below). However it is also possible to change this default. For example, if you are intending to use bi-stable light switches throughout your home, instead of setting individual config for each input, you can simply set the `defaultInputType` to `switch` and be done with it.
+By default all inputs are initialised as type `switch`. Individual inputs can then be configured as required ([see below](./state-monitor-esp32.md#input-config)). However it is also possible to change this default. For example, if you are intending to use push buttons throughout your home, instead of setting individual config for each input, you can simply set the `defaultInputType` to `button` and be done with it.
 
-|Key|Mandatory|Value|
-|---|---------|-----|
-|`defaultInputType`|Optional|Either `button`, `contact`, `rotary`, `switch` or `toggle`|
+|Key|Value|  
+|:--|:----|
+|`defaultInputType` |Either&nbsp;`button`,&nbsp;`contact`,&nbsp;`press`,&nbsp;`rotary`,&nbsp;`switch`&nbsp;or&nbsp;`toggle`|

```

Comment:
What are these `&nbsp` chars in here? Spaces? Can we remove?

Reply:
the table doesn't render nicely with spaces, `&nbsp` prevents line break and distributes column width nicer. Didn't find a better solution

Reply:
Yeah I have seen tables render a bit weird - but I think it is better to keep the source code clean and worry about fixing the rendering later? Just that we don't do this anywhere else and would be a pain to retrofit this fix to all tables.